tests/unit/test_cli.py

```diff
@@ -7,15 +7,26 @@
 from cou.steps import UpgradeStep
 
 
-def test_parse_args():
-    args = ["--dry-run", "--log-level", "DEBUG", "--interactive", "true"]
+@pytest.mark.parametrize(
+    "args, expected_results",
+    [
+        (["--run", "--log-level", "DEBUG", "--non-interactive"], [True, "DEBUG", True]),
+        (["--run", "--log-level", "warning"], [True, "WARNING", False]),
+        (["--log-level", "debug"], [False, "DEBUG", False]),
+    ],
+)
+def test_parse_args(args, expected_results):
     parsed_args = parse_args(args)
-    assert parsed_args.dry_run
-    assert parsed_args.loglevel == "DEBUG"
-    assert parsed_args.interactive
+    expected_run = expected_results[0]
+    expected_loglevel = expected_results[1]
+    expected_non_interactive = expected_results[2]
```

Comment:
not-blocking: What is purpose to use single `expected_results` and than use it only in single place to unpack it to multiple variable. Consider to use ` expected_run`, `expected_loglevel` and `expected_non_interactive` as inputs for this tests.

Reply:
Solved